server/src/controllers/addJob.ts

```diff
@@ -0,0 +1,32 @@
+import { AuthRequest } from '../interfaces';
+import { Job } from '../models';
+import { addJobValidation } from '../validation';
+
+const addJob = async (req: AuthRequest) => {
```

Comment:
create a `job.ts` file at controller and combine both `getJobs` and `addJob` and `search-jobs` at the some file